arangod/Aql/Functions.cpp

```diff
@@ -8848,6 +8855,132 @@ AqlValue Functions::MakeDistributeGraphInput(arangodb::aql::ExpressionContext* e
   return AqlValue{input};
 }
 
+template <typename F>
+AqlValue calc_function(arangodb::aql::ExpressionContext* expressionContext,
```

Comment:
* camelCase
* use meaningful function name, e.g. `decayFuncImpl`
* move to annonymous namespace